liveRecorder/Tasks/TaskRunner.py

```diff
@@ -227,3 +227,17 @@ def safe_move(self, src, dst):
 
         except Exception as e:
             self.logger.error("Failed to move %s to %s : %s \n %s", src, dst, str(e), traceback.format_exc())
+    def getSorterFileList(self, src_dir):
+        file_list = os.listdir(src_dir)
+        file_list_with_ctime = []
+        for path in file_list:
```

Comment:
usually i prefer:
file_list_with_ctime = [ (path,os.stat(src_dir + '/' + path).st_ctime) for path in file_list]